tutorials/introduction.ipynb

```diff
@@ -17,12 +17,18 @@
+    }
+   ],
    "source": [
-    "#!pip install asent\n",
-    "#!python -m spacy download en_core_web_lg"
+    "!pip install asent\n",
+    "!python -m spacy download en_core_web_lg # Do we need the large model here? Getting people into the tutorial quickly seems beneficial."
```

Comment:
Ensure consistency, but I agree it might be better to use a smaller one

Changing for now to remove the comment - I will try to change to a smaller model.
```suggestion
    "!python -m spacy download en_core_web_lg"
```